Sun May 13 19:39:35 FET 2012  Sergei Trofimovich <slyfox@community.haskell.org>
  * loosen deps on pretty and mtl
diff -rN -u old-xcb-types/xcb-types.cabal new-xcb-types/xcb-types.cabal
--- old-xcb-types/xcb-types.cabal	2012-05-13 19:40:35.698961327 +0300
+++ new-xcb-types/xcb-types.cabal	2012-05-13 19:40:35.860959303 +0300
@@ -30,7 +30,7 @@
 
 Library
 
- Build-depends: base == 4.*, xml == 1.3.*, pretty == 1.0.*, mtl == 2.0.*
+ Build-depends: base == 4.*, xml == 1.3.*, pretty >= 1.0 && < 1.2, mtl >= 2.0 && < 2.2
  Exposed-modules: Data.XCB,
                   Data.XCB.Types,
                   Data.XCB.Pretty,
